api/tests/views_tests.py

```diff
@@ -466,6 +466,23 @@ def test_201_new_user_created_and_202_user_exists(
         assert responses.assert_call_count(self.fxa_verify_path, 2) is True
         assert responses.assert_call_count(FXA_PROFILE_URL, 1) is True
 
+    @responses.activate()
+    def test_failed_profile_fetch_for_new_user_returns_500(self):
+        user_token = "user-123"
+        self._setup_client(user_token)
+        now_time = int(datetime.now().timestamp())
+        exp_time = (now_time + 60 * 60) * 1000
+        _setup_fxa_response(200, {"active": True, "sub": self.uid, "exp": exp_time})
```

Comment:
quibble (non-blocking): I thought this was unneccessary at first, because we add the response for `FXA_PROFILE_URL` after this. But I see that this `_setup_fxa_response` might be better-named `_setup_fxa_introspection_response`?

Reply:
Filed [MPP-3527](https://mozilla-hub.atlassian.net/browse/MPP-3527) and added a `TODO` comment